### DISCUSSION TITLE
PWA module

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3635] [Extensions & Modules]
I found this module which enables PWA functionality in Magento

It seems to work in OpenMage too.

What do you think?

https://github.com/meanbee/magento-meanbee-pwa

https://www.youtube.com/watch?v=b31hLf1S6BY

**Comment by m-overlund** [comment]
I've tried it once.
As i remember there's some multistore issues.
Didn't get it to work properly, at that time
